src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -117,7 +94,7 @@ public void onEnable() {
             new CreateUserListener(this)
         ).forEach(listener -> Bukkit.getPluginManager().registerEvents(listener, this));
 
-        Bukkit.getConsoleSender().sendMessage(ChatUtils.color("&7[EternalCore] Successfully loaded EternalCore in " + (System.currentTimeMillis() - startTime) + "ms"));
+        getLogger().info(ChatUtils.color("&7[EternalCore] Successfully loaded EternalCore in " + (System.currentTimeMillis() - startTime) + "ms"));
```

Comment:
getLoggery z bukkita są giga ussless, Bukkit#getConsoleSender jest o wiele lepszy

Reply:
A nie lepiej użyć getLogger() z instancji EternalCore i w plugin.yml zastosować prefix do logów by pozbyć się `&7[EternalCore]` które wydaję się obecnie być useless?

Reply:
Ta zmiana jest jak najbardziej dobra, logger ma po to swoje zadanie i levele logowania, też programy nasłuchujące loggera nie będą widzieć logów z eternala.
Jak dla mnie Hyd3r1 ma rację.

Reply:
Z tego co wiem to logger automatycznie dodaje prefixy.

Reply:
> 

Z tego co widzę nie posiadamy pliku plugin.yml :(